content/source/docs/cloud/sentinel/mock.html.md

```diff
@@ -267,9 +267,7 @@ The following factors can prevent you from generating mock data:
   produced via mock generation.
 * The run has not progressed past the planning stage, or did not create a plan
   successfully.
-* The run has been in a terminal state, such as applied or discarded, longer
-  than seven days. At this point, the data necessary to generate the mocks is no
-  longer available.
+* The run had progressed past the planning stage prior to July 23, 2021. Prior to this date, JSON plans were only kept for 7 days.
```

Comment:
Again, just want it to be active voice :-)

```suggestion
* The run progressed past the planning stage prior to July 23, 2021. Prior to this date, Terraform Cloud only kept JSON plans for 7 days.
```

Reply:
I would rather see us use this:
```
The run progressed past the planning stage prior to July 23, 2021 and more than seven days had passed. (After that date, Terraform Cloud began permanently keeping the JSON plans that mock generation requires.)
```

Reply:
I like your edit except - I think let's omit the parenthesis and say "the JSON plans required for mock generation". If we do that, I'm sold!